src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -104,6 +104,12 @@ void Push(StackValue value)
         {
             FatalCheck(_stackTop < _maxStack, VerifierError.StackOverflow);
 
+            if (HasPendingPrefix(Prefix.ReadOnly))
```

Comment:
Note that read-only values can be produced even without the readonly prefix, e.g. unbox will produce the readonly value. It may be nice to add TODO about it or something.